jetty-websocket/websocket-servlet/src/main/java/org/eclipse/jetty/websocket/servlet/ServletUpgradeRequest.java

```diff
@@ -221,13 +196,6 @@ public String getQueryString()
         return this.queryString;
     }
 
-    /**
-     * Return a {@link SocketAddress} for the remote socket.
-     * <p>
-     * Warning: this can cause a DNS lookup
-     *
-     * @return the remote socket address
-     */
     public SocketAddress getRemoteSocketAddress()
```

Comment:
Restore javadoc.
Add mention that SocketAddress is not garunteed to be an IPv4 or IPv6, it could be something else, like UnixSocketAddress.